internal/cli/setup/cluster_setup.go

```diff
@@ -31,6 +32,7 @@ import (
 
 var ErrNoRegions = errors.New("no regions found for the cloud provider")
 var ErrNoVersions = errors.New("no mongodb versions found for the cloud provider")
+var deprecatedClusterWideScalingMessage = "If you require more flexibility in shard scaling, consider using --autoScalingMode independentShardScaling."
```

Comment:
I believe this message lacks sufficient context. Maybe we should add something like "Defaulting to X, if you require more flexibility..."

Reply:
cooll thanks! i'm adding a statement saying that clusterWideMode has been detected (since we also want to send the warn if users set the flag)